www-apps/phpsysinfo/phpsysinfo-3.2.10.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp
+
+DESCRIPTION="phpSysInfo is a nice package that will display your system stats via PHP"
```

Comment:
Please no package name in the `DESCRIPTION`

Reply:
Fixed.